MapboxMobileEvents/Categories/NSUserDefaults+MMEConfiguration.m

```diff
@@ -285,11 +287,15 @@ - (NSString *)mme_userAgentString {
     static dispatch_once_t onceToken;
     dispatch_once(&onceToken, ^{
 
-        userAgent = [NSString stringWithFormat:@"%@/%@ (%@; v%@)",
+        userAgent = [NSString stringWithFormat:@"%@/%@ (%@; v%@; %@ %@; %@)",
             NSBundle.mme_mainBundle.infoDictionary[(id)kCFBundleNameKey] ?: NSBundle.mme_mainBundle.bundlePath.lastPathComponent.stringByDeletingPathExtension,
             NSBundle.mme_mainBundle.mme_bundleVersionString,
             NSBundle.mme_mainBundle.bundleIdentifier,
-            NSBundle.mme_mainBundle.infoDictionary[(id)kCFBundleVersionKey]];
+            NSBundle.mme_mainBundle.infoDictionary[(id)kCFBundleVersionKey],
+            UIDevice.currentDevice.systemName,
```

Comment:
[mme_osVersion](https://github.com/mapbox/mapbox-events-ios/blob/7a943206b647b27b736a719f39dacf17f520377e/MapboxMobileEvents/Categories/NSProcessInfo%2BSystemInfo.m#L64) would be nice here to remove the UIDevice dependency

Reply:
Although I can see the benefit of an abstraction from UIDevice. 

The SDK at present does not support MacOS either as a target. I wonder if it's necessary to get in just before a release? Could this perhaps wait for a follow up release which has the necessary targets to require implementation?

Reply:
Updated